### PR TITLE
Codeclimate prepare

### PR DESCRIPTION
**Pull request by AndrewRayCode**
Codeclimate prepare

```
$ codeclimate help
    prepare
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,11 +77,13 @@ A list of available commands is accessible by running `codeclimate` or
...
 
 Available commands:
-    analyze [-f format] [-e engine(:channel)] [--dev] [path]
+    analyze [-f format] [-e engine[:channel]] [path]
     console
     engines:install
     engines:list
-    help
+    help [command]
+    prepare [--allow-internal-ips]
```
Comment (AndrewRayCode) [inline README.md]: codeclimate prepare

File: README.md [hunk 2]
```diff
@@ -77,11 +77,13 @@ A list of available commands is accessible by running `codeclimate` or
 $ codeclimate help
 
 Available commands:
-    analyze [-f format] [-e engine(:channel)] [--dev] [path]
+    analyze [-f format] [-e engine[:channel]] [path]
```
Comment (wfleming) [inline README.md]: I think we'd like to keep the `--dev` flag documented here, please.

File: README.md [hunk 3]
```diff
@@ -77,11 +77,13 @@ A list of available commands is accessible by running `codeclimate` or
...
     console
     engines:install
     engines:list
-    help
+    help [command]
+    prepare [--allow-internal-ips]
+    test <engine_name>
```
Comment (wfleming) [inline README.md]: This is not really a supported command, despite the code still existing for it. We don't recommend engine authors use it, so I think we'd like to keep it out of the README.

Comment (AndrewRayCode) [reply README.md]: what isn't? `test` ?

Comment (wfleming) [reply README.md]: Yes. `test` is not encouraged to be used, and we'd prefer to leave it undocumented.

Comment (AndrewRayCode) [reply README.md]: codeclimate prepare

File: README.md (hunk 2 above)
Comment (AndrewRayCode) [reply README.md]: codeclimate prepare

**Comment by AndrewRayCode** [comment]
@wfleming codeclimate prepare

**Comment by wfleming** [comment]
Thank you for the docs corrections, @AndrewRayCode. If you can sign the CLA I'll be happy to merge this.

**Comment by AndrewRayCode** [comment]
codeclimate prepare 😄

**Comment by wfleming** [comment]
codeclimate prepare, indeed.

**Pull request merged**